qemu/tests/hpt_miscellaneous.py

```diff
@@ -118,4 +118,11 @@ def run(test, params, env):
         session = vm.reboot(session)
         vm.verify_alive()
         verify_hpt(test, params, session, hpt_default)
-    vm.verify_kernel_crash()
+    if "migration" in params.get("sub_type"):
```

Comment:
How about `params.get("sub_type") = "migration"`.

Reply:
Sure and will update it